src/PowerSystems.jl

```diff
@@ -139,6 +139,7 @@ export SteamTurbineGov1
 # Converter Exports
 export Converter
 export AverageConverter
+export REGCA1
```

Comment:
```suggestion
export NortonEquivalentConverter
```

Reply:
Not possible since REGCA is not only the NortonEquivalentConverter. We could do maybe `RenewableEnergyConverterTypeA` or `REConverterA`, something like that?

Reply:
RenewableEnergyConverterTypeA